src/config/index.ts

```diff
@@ -42,11 +43,11 @@ export const Config = {
   },
   worldpay: {
     testnet: {
-      apiUrl: loadEnvironmentVariable('WORLDPAY_API_URI'),
-      entityRef: loadEnvironmentVariable('WORLDPAY_ENTITY_REF'),
-      username: loadEnvironmentVariable('WORLDPAY_USERNAME'),
-      password: loadEnvironmentVariable('WORLDPAY_PASSWORD'),
-      accessCheckoutId: loadEnvironmentVariable('WORLDPAY_ACCESS_CHECKOUT_ID'),
+      apiUrl: loadEnvironmentVariable('WORLDPAY_API_URI', ''),
+      entityRef: loadEnvironmentVariable('WORLDPAY_ENTITY_REF', ''),
+      username: loadEnvironmentVariable('WORLDPAY_USERNAME', ''),
+      password: loadEnvironmentVariable('WORLDPAY_PASSWORD', ''),
+      accessCheckoutId: loadEnvironmentVariable('WORLDPAY_ACCESS_CHECKOUT_ID', ''),
```

Comment:
The defaults should not be empty. 
```suggestion
      apiUrl: loadEnvironmentVariable('WORLDPAY_API_URI'),
      entityRef: loadEnvironmentVariable('WORLDPAY_ENTITY_REF'),
      username: loadEnvironmentVariable('WORLDPAY_USERNAME'),
      password: loadEnvironmentVariable('WORLDPAY_PASSWORD'),
      accessCheckoutId: loadEnvironmentVariable('WORLDPAY_ACCESS_CHECKOUT_ID'),
```